ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -153,6 +153,10 @@ class CellEditorOverlay extends Component {
       ..._.mapValues(queryModifiers, qm => this.queryStateReducer(qm)),
     }
 
+    const querySavable = query =>
```

Comment:
Oof, this spelling of `savable` (as opposed to `saveable`) makes me cringe. Apparently it's legitimate 😱 

But on a more pertinent note, I suggest `queryIsSavable` or `isQuerySavable` to be consistent with the `isSavable` semantics.